energyplus_regressions/diffs/table_diff.py

```diff
@@ -435,8 +484,19 @@ def table_diff(thresh_dict, inputfile1, inputfile2, abs_diff_file, rel_diff_file
                                table_not_in_1, table_not_in_2)
             continue
 
+        # create a list of order-dependent table uheading keys, tables that include these keys in the name
+        # these will use strict row order enforcement
+        row_order_dependent_table_keys = ['Monthly', 'Topology']
```

Comment:
Here we identify any tables that should enforce row order when doing diffs.  Things like topology or time-based should be added here.  This can be adjusted as more are identified.